forge-gui/res/cardsfolder/upcoming/poetic_ingenuity.txt

```diff
@@ -0,0 +1,11 @@
+Name:Poetic Ingenuity
+ManaCost:2 R
+Types:Enchantment
+T:Mode$ AttackersDeclared | ValidAttackers$ Dinosaur.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever one or more Dinosaurs you control attack, create that many Treasure tokens.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ c_a_treasure_sac
+SVar:X:TriggerObjectsAttackers$Valid Dinosaur.YouCtrl
```

Comment:
just $Amount for correctness
please also fix this one where you copied it from
let me know if you need explanation why

Reply:
I got this from Basri Ket. I guess `Amount ` should be enough since it will get the valid from line 4 ?

Reply:
yeah, TriggerObjectsAttackers is already filtered, so Amount should be enough